### PR TITLE
validation created

### PR DESCRIPTION
**Pull request by Reona001**
Could you check my project model validation

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -1,3 +1,12 @@
 class Project < ApplicationRecord
   belongs_to :user
+  validates :project_name, presence: true, uniquness: true
+  validates :description, presence: true, length: { maximum: 2000 }
+  validates :people_required, presence: true, inclusion: { in: 1..100 }
+  validates :location, presence: true
+  validates :start_date, presence: true
+  validates :end_date, presence: true
+  validates :reward_eaned, presence: true
```
Comment (nibab-boo) [inline app/models/project.rb]: Line 9 has a typo

File: app/models/project.rb [hunk 2]
```diff
@@ -1,3 +1,12 @@
 class Project < ApplicationRecord
   belongs_to :user
+  validates :project_name, presence: true, uniquness: true
```
Comment (switchloops) [inline app/models/project.rb]: typo - uniqueness

File: app/models/project.rb (hunk 1 above)
Comment (switchloops) [inline app/models/project.rb]: typo - reward_earned

**Review by switchloops — CHANGES_REQUESTED** [review]

**Review by nibab-boo — CHANGES_REQUESTED** [review]
Typo

**Pull request merged**